crates/ruff_python_formatter/tests/snapshots/black_compatibility@py_310__pattern_matching_extras.py.snap

```diff
@@ -371,12 +368,9 @@ match something:
     case 2 as b, 3 as c:
         pass
 
-    case (
-        4 as d,
-        5 as e,
-        NOT_YET_IMPLEMENTED_PatternMatchOf | (y) as g,
-        *NOT_YET_IMPLEMENTED_PatternMatchStar,
-    ):
+    case 4 as d, (5 as e), (
+        NOT_YET_IMPLEMENTED_PatternMatchOf | (y) as g
+    ), *NOT_YET_IMPLEMENTED_PatternMatchStar:
```

Comment:
Unrelated but shouldn't we consider the magic trailing comma and keep each pattern on a separate line?

Reply:
We probably should